### PR TITLE
Added screenshot showing pages in README.md

### PR DESCRIPTION
**Pull request by joejoe31b**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Kimai - jQuery Mobile Interface
 ===============================
 
+![Screenshots](http://i.imgur.com/uP8zGKv.png)
```
Comment (simonschaufi) [inline README.md]: no screenshots from third party server!

Comment (joejoe31b) [inline README.md]: Where would you prefer me to host it?

Comment (simonschaufi) [inline README.md]: how about adding it to the repository and reference it there?

**Comment by joejoe31b** [comment]
Created a branch for screenshots, as it is unnecessary to have in the master. Commit 6b4b63a.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Kimai - jQuery Mobile Interface
 ===============================
 
+![Screenshots](/../images/screenshots.png?raw=true)
```
Comment (simonschaufi) [inline README.md]: I don't see the added image in that pull request. Did you forget to add it?

**Comment by kevinpapst** [comment]
Please wait before doing any more work.
I will create a small page instead, as GitHub is place for developer and not end-consumer.
I'll let you know soon how to proceed.

File: README.md (hunk 2 above)
Comment (joejoe31b) [inline README.md]: I created a separate branch for the screenshots. Commit 6b4b63a.

Comment (simonschaufi) [inline README.md]: Sorry to say that but you totally messed up that commit. Anyway, wait for Kevin.

Comment (joejoe31b) [inline README.md]: My apologies. I had no idea how else to do it, and are still getting the hang of this.

For future reference, how was I supposed to do it? Was I supposed to create an orphan branch (which I only just found out about now)?

**Comment by kevinpapst** [comment]
I created a page for Kimai Mobile: http://www.kimai.org/kimai-mobile/
It can be found in this branch: https://github.com/kimai/kimai-mobile/tree/gh-pages
Please add screenshots in this directory: https://github.com/kimai/kimai-mobile/tree/gh-pages/kimai
and link the screenshot in index.html

**Comment by kevinpapst** [comment]
Oh and please do that in a new Pull request. Thanks @joejoe31b !

File: README.md (hunk 2 above)
Comment (simonschaufi) [inline README.md]: Just stay on the same branch as you did the pull request from, make your changes, commit and push like before. No need to complicated orphan branches (no idea what they are for)

**Pull request closed without merging**